src/ucp/rndv/rndv.c

```diff
@@ -566,21 +564,16 @@ UCS_PROFILE_FUNC_VOID(ucp_rndv_get_completion, (self), uct_completion_t *self)
 {
     ucp_request_t *rndv_req = ucs_container_of(self, ucp_request_t,
                                                send.state.uct_comp);
+    ucs_status_t status     = rndv_req->send.state.uct_comp.status;
```

Comment:
imo, better leave it as it was before and initialize status when it is really needed

Reply:
yes, it is leftover - done